spec.html

```diff
@@ -11832,14 +11832,29 @@ <h1>Execution Contexts</h1>
             Generator
           </td>
           <td>
-            The Generator that this execution context is evaluating.
+            The Generator that is associated with this execution context.
           </td>
         </tr>
       </table>
     </emu-table>
-    <p>In most situations only the running execution context (the top of the execution context stack) is directly manipulated by algorithms within this specification. Hence when the terms “LexicalEnvironment”, and “VariableEnvironment” are used without qualification they are in reference to those components of the running execution context.</p>
+    <p>To express the semantics of generators, this document uses steps of the form “<dfn id="transfer-control" variants="transfers control,transfer of control">Transfer control</dfn> from _aContext_ to _bContext_, passing _completion_”, where _aContext_ and _bContext_ are execution contexts and _completion_ is a Completion Record. This means:</p>
+    <ul>
+      <li>Execution of the current algorithm (the one with the “Transfer control” step) is suspended, and a representation of that suspended execution is saved in the code evaluation state of _aContext_.</li>
```

Comment:
```suggestion
      <li>Execution of the current algorithm (the one with the “transfer control” step) is suspended, and a representation of that suspended execution is saved in the code evaluation state of _aContext_.</li>
```

Consistent with the below reference.

Reply:
Agreed there's an inconsistency, but I fixed it in the other direction, capitalizing the later references.